### DISCUSSION TITLE
感觉alovajs和nuxt3好像

### DISCUSSION THREAD

**Discussion by wintsa123** [alovajs/alova#184] [General]
nuxt3的请求有个useAsyncData的可以设定key，结合nuxt3的状态管理，usestate，就好像alova的数据缓存功能

**Comment by JOU-amjs** [comment]
是啊，比如nuxt、react-query、swr他们都有数据缓存功能，alova也是在他们之后才出来的，alova在这基础上改良了缓存功能，key是自动管理。的
